admin/views/class-view-utils.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @package WPSEO\Admin\Views
+ */
+
+/**
+ * Class Yoast_View_Utils
+ */
+class Yoast_View_Utils {
+	/**
+	 * Yoast_View_Utils constructor.
+	 */
+	public function __construct() {
+		$this->form = Yoast_Form::get_instance();
+	}
+
+	/**
+	 * Shows the search appearance settings for a post type.
+	 *
+	 * @param string|object $post_type The post type to show the search appearance settings for.
+	 */
+	public function show_post_type_settings( $post_type ) {
+		if ( ! is_object( $post_type ) ) {
+			$post_type = get_post_type_object( $post_type );
+		}
+		/* translators: %s expands to the post type name */
+		$this->form->index_switch( 'noindex-' . $post_type->name, sprintf( __( 'Show %s in search results?', 'wordpress-seo' ), '<strong>' . $post_type->labels->name . '</strong>' ) );
+		$this->form->textinput( 'title-' . $post_type->name, __( 'Title template', 'wordpress-seo' ), 'template posttype-template' );
+		$this->form->textarea( 'metadesc-' . $post_type->name, __( 'Meta description template', 'wordpress-seo' ), array( 'class' => 'template posttype-template' ) );
+		$this->form->toggle_switch( 'showdate-' . $post_type->name, array(
```

Comment:
Maybe its nice to wrap this in its own method. This method can reused for the `toggle_switch` below